record-two/index.html.in

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html ng-app="chaise.record">
+<head>
+    <title>Record</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=10; IE=9; IE=8; IE=7; IE=EDGE" />
+    %ASSETS%
+    <!--
+        IE8 support, see AngularJS Internet Explorer Compatibility http://docs.angularjs.org/guide/ie
+        For Firefox 3.6, you will also need to include jQuery and ECMAScript 5 shim
+    -->
+    <!--[if lt IE 9]>
+        <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.js"></script>
+        <script src="http://cdnjs.cloudflare.com/ajax/libs/es5-shim/2.2.0/es5-shim.js"></script>
+        <script>
+            document.createElement('ui-select');
+            document.createElement('ui-select-match');
+            document.createElement('ui-select-choices');
+        </script>
+    <![endif]-->
+</head>
+<body>
+    <navbar></navbar>
+
+    <div id="entity-title">{{ table.displayname }}</div>
+
+    <!-- defined on $rootScope in run function -->
+    <record-display record="record" table="table"></record-display>
```

Comment:
What's the `table` binding for? If it doesn't change, maybe a one-time bind here is appropriate.